FordLabs/PeopleMover
# Testing filter of location and product tags in cypress

## Issue
Resolves #455 

## What was done
- [x] Added a test for filtering by location
- [x] Added a test for filtering by product tag

## How to test
1. Run the cypress tests, verify that they are accurately testing the filter logic

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
